app/components/documents/document-details-panel.hbs

```diff
@@ -179,23 +179,39 @@
       </Auk::KeyValue>
       {{#if (and this.isSignaturesEnabled (await (this.canViewSignedPiece)))}}
         {{#if @piece.signedPiece.file}}
-          <Auk::KeyValue
-            @key={{t "signed-document"}}
-          >
-            <AuLinkExternal
-              href={{await @piece.signedPiece.namedDownloadLinkPromise}}
-              @skin="primary"
-              @icon="download"
-              @iconAlignment="left"
-              download
-            >
-              {{@piece.signedPiece.name}}.{{@piece.signedPiece.file.extension}}
-            </AuLinkExternal>
-            <AccessLevelPill 
-              @accessLevel={{await @piece.signedPiece.accessLevel}} 
-              @isEditable={{false}}
-            />
-          </Auk::KeyValue>
+          <div class="auk-key-value-item">
+            <div class="auk-key-value-item__label">{{t "signed-document"}}</div>
+            <div class="auk-key-value-item__value au-u-flex au-u-flex--vertical-center au-u-flex--spaced-small">
+              <AuLinkExternal
+                href={{await @piece.signedPiece.namedDownloadLinkPromise}}
+                @skin="primary"
+                @icon="download"
+                @iconAlignment="left"
+                download
+              >
+                {{@piece.signedPiece.name}}.{{@piece.signedPiece.file.extension}}
+              </AuLinkExternal>
+              <AccessLevelPill 
+                @accessLevel={{await @piece.signedPiece.accessLevel}} 
+                @isEditable={{false}}
+              />
+              <AuDropdown
```

Comment:
I think a `{{#if (user-may "remove-signatures")}}` would be appropriate here. Now this is only checked by `this.currentSession.may('manage-signatures');` in `isSignaturesEnabled`.

If we have a separate permission for removal, might as well use it consistently.